### PR TITLE
Added a sweeping kitchen 

### PR DESCRIPTION
**Pull request by Tanosimboangy**
Please team if you have a comment, write down here!

### CODE REVIEW

**Review by Fodilahy-mena — APPROVED** [review]

**Comment by Fodilahy-mena** [comment]
Thank you Mr jacquit

File: Recipe.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# CASSAVA WITH COCONUT AND VANILLA
```
Comment (Fodilahy-mena) [inline Recipe.md]: Oops looks like this file shouldn't be here!

Comment (Fodilahy-mena) [inline Recipe.md]: Ohooo men,replay my message!

Comment (Fodilahy-mena) [inline Recipe.md]: Maybe you were so hungry and you want to eat BABOFOKA

**Review by Fodilahy-mena — CHANGES_REQUESTED** [review]

File: Recipe.md (hunk 1 above)
Comment (Tanosimboangy) [reply Recipe.md]: Thanks my friend for reminding me that, I'm gonna make a change now to fix this !

Comment (Tanosimboangy) [reply Recipe.md]: Yes ,I've replied it !

Comment (Tanosimboangy) [reply Recipe.md]: No ! I'm still fine ! It was just a mistake.

File: sweeping-the-kitchen.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# Sweeping the kitchen
```
Comment (Fodilahy-mena) [inline sweeping-the-kitchen.md]: Hi jack! your work looked good but only has confusing list which reminds your babofoka

**Review by Fodilahy-mena — CHANGES_REQUESTED** [review]

File: sweeping-the-kitchen.md (hunk 2 above)
Comment (Tanosimboangy) [reply sweeping-the-kitchen.md]: BE serious man!Let's not talk about Babofoka anymore

File: Recipe.md (hunk 1 above)
Comment (vakodrazan) [reply Recipe.md]: I think you should not write all  of your title.

Comment (vakodrazan) [reply Recipe.md]: I think you should not write into capital letter all of your title.

**Pull request closed without merging**